packages/wonder-blocks-modal/src/themes/default.ts

```diff
@@ -15,6 +15,8 @@ const theme = {
     },
     border: {
         radius: tokens.border.radius.medium_4,
+        width: tokens.border.width.thin,
+        color: tokens.color.blue,
```

Comment:
**note:** Added these two new component tokens to be used in the dismiss button for the focus ring